tests/test_builders/test_build_gettext.py

```diff
@@ -170,6 +170,15 @@ def test_gettext_template_msgid_order_in_sphinxpot(app):
         flags=re.DOTALL)
 
 
+@pytest.mark.sphinx('gettext', testroot='gettext-custom-output-template')
+def test_gettext_custom_output_template(app):
+    app.build(force_all=True)
+    assert (app.outdir / 'index.pot').is_file()
+
+    result = (app.outdir / 'index.pot').read_text(encoding='utf8')
+    assert "EVEN MORE DESCRIPTIVE TITLE" in result
```

Comment:
@jmbowman this test case passes without any changes from your branch in the `sphinx/builders/gettext.py` file - are you missing some additional test assertions?

Reply:
Hold on; this may have been a mistake during my testing of the changes.  I'm double-checking what I did here..

Reply:
Ok, my mistake.  I had run `pytest tests/test_intl/test_intl.py` as a basic verification -- but that's the wrong test file.

Without the updates to the builder code, `pytest tests/test_builders/test_build_gettext.py` -- the correct test module -- does fail, as expected.